google/cloud/functions/internal/setenv.cc

```diff
@@ -0,0 +1,56 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "google/cloud/functions/internal/setenv.h"
+#ifdef _WIN32
+// We need _putenv_s()
+#include <stdlib.h>
+#else
+// On Unix-like systems we need setenv()/unsetenv(), which are defined here:
+#include <stdlib.h>  // NOLINT
```

Comment:
Why do we need the `NOLINT`?

That said ...
```
#if FOO
#include BAR
#else
#include BAR
#endif
```
look weird.

Reply:
Fixed, I think we need to fix google-cloud-cpp too where I cut&pasted this from.